src/EventTeachingService.cpp

```diff
@@ -32,10 +48,7 @@ Processed EventTeachingService::handleMessage(unsigned int opc, CANFrame *msg)
 
     if (nn == module_config->nodeNum)
     {
-      bLearn = true;
-      // DEBUG_SERIAL << F("> set lean mode ok") << endl;
-      // set bit 5 in parameter 8
-      bitSet(controller->_mparams[8], 5);
+     enableLearn();
```

Comment:
Please put one more space in here so you have a nice indentation.

Reply:
OK